src/doing/utils.py

```diff
@@ -372,7 +372,7 @@ def get_current_pr_id() -> int:
         console.print("Could not find a PR associated with the current branch: " + get_git_current_branch())
         sys.exit(1)
     else:
-        return result[0].get("pullRequestId")
+        return result[0].get("pullRequestId")  # type: ignore[no-any-return]
```

Comment:
Similar issue as above. We might want to assert this is an `int`, but for now I just made `mypy` ignore this issue.